packages/convertwithmoss/PKGBUILD

```diff
@@ -0,0 +1,33 @@
+# Maintainer: OSAMC <https://github.com/osam-cologne/archlinux-proaudio>
+# Contributor: Christopher Arndt <osam -at- chrisarndt -dot- de>
+# Contributor: redtide <redtid3 -at- gmail -dot- com>
+
+_name=ConvertWithMoss
+pkgname=${_name,,}
+pkgver=6.2.0
+pkgrel=1
+pkgdesc='A tool for converting multi-sample from one format to another'
+url='https://mossgrabers.de/Software/ConvertWithMoss/ConvertWithMoss.html'
+arch=(any)
+license=(LGPL)
+depends=('java-runtime>=16' java-openjfx)
```

Comment:
The package doesn't install (and build) on aarch64 because there is no `java-openjfx` package, should we just set arch to `x86_64` then?

Reply:
Turns out, it isn't needed at all, an architecture-specific package of javafx is apparently downloaded during build and included in the package.

Reply:
I see, thanks for looking into that!